httpstats/metrics.go

```diff
@@ -197,6 +202,15 @@ func (m *metrics) observeRequest(req *http.Request, op string, bodyLen int) {
 	m.http.method = req.Method
 	m.http.protocol = req.Proto
 	m.http.transferEncoding = transferEncoding
+
+	if m.options.pathObserved {
+		path := req.URL.Path
+
```

Comment:
the [docs](https://golang.org/pkg/net/url/) for URL.Path mention that sometimes it includes the leading slash and sometimes not. Added this for consistency